api.py

```
@@ -0,0 +1,713 @@
+        persona_name=persona_name,
+        api_key=request.meeting_baas_api_key,
+        recorder_only=request.recorder_only,
+        bot_image=request.bot_image,
+        entry_message=request.entry_message,
+        extra=request.extra,
+    )
+
+    if meetingbaas_bot_id:
+        # Start the Pipecat process for this bot
+        pipecat_websocket_url = f"ws://localhost:8766/pipecat/{bot_client_id}"
+        process = start_pipecat_process(
```

Ruff (0.8.2)

Local variable `process` is assigned to but never used

Remove assignment to unused variable `process`

(F841)